De/coupled trap L/D from cavity L/D

mohiuddinrazu opened the pull request:
### New Features
Trap L/D can now be decoupled from cavity L/D. Effective volume is calculated from trap volume.

### Tests Completed
None (could not get docker to cooperate)

taliaweiss commented:
@mohiuddinrazu , would you be willing to modify this so that the input parameter is the trap_length instead of trap_L_over_D? The reason is that "trap L/D" sounds like it could be [trap length / trap diameter]. But really, the relevant parameter here, and the one you're using, is [trap length / cavity parameter]. To avoid confusion about this, I think it would be clearest to simply have trap_length be the parameter provided in the config files and used in mermithid.

Also, could you please rename L_over_D to cavity_L_over_D, so that it's clear what this parameter is?

Thank you!

mohiuddinrazu commented:
@taliaweiss Good points. I've modified the code to take trap length as an input (or just use cavity length if user wants to). Also renamed L_over_D to cavity_L_over_D. Let me know if you have more comments, thank you!

taliaweiss commented:
Great, thank you! Those recent changes look good. I'll do a fuller read-through soon.

taliaweiss commented:
Hi @mohiuddinrazu , I read through all of your code, which looks great, then made a sensitivity vs. density plot with your branch. Two bugs came up during running – I fixed the bugs and pushed the changes. I also resolved two merge conflict that arose due to changes I made to the base branch.

Everything looks good to me now! Feel free to check over my changes. After that, I will merge the branch and close the PR.

mohiuddinrazu commented:
Thank you @taliaweiss for reviewing the code and running tests. Your changes look good and runs on my Docker. 
I think it's ready to be merged now.
